src/kgcl_schema/schema/kgcl.yaml

```diff
@@ -625,7 +627,7 @@ classes:
     slot_usage:
       change_description:
         string_serialization: obsoleting {about_node}
-      associated change set:
+      associated_change_set:
         is_a: change_set
```

Comment:
Is it necessary to say `is_a: change_set` here when it's already part of the slot definition below?